pypeit/tests/test_scripts.py

```diff
@@ -291,5 +294,8 @@ def test_identify():
     os.remove('wvarxiv.fits')
     os.remove('wvcalib.fits')
 
+    # Move old waveid.ascii back to keep git clean
+    shutil.move(data_path('waveid.ascii.tmp'), data_path('waveid.ascii'))
+
```

Comment:
this one is my fault.  Any solution you can find,
I approve.

Reply:
There's a filename parameter in identify.py::save_IDs that is never changed from its default in any of the current invocations. I think the easiest thing for now is just to rename waveid.ascii to something like waveid_test.ascii so that there's no conflict in the first place.

Reply:
Personally, I'm fine if you just leave the PR as is and punt on this one.

Reply:
agreed!